Adyen/UI/Form/Items/Value Pickers/FormRegionPickerItem.swift

```diff
@@ -16,15 +16,26 @@ public typealias RegionPickerItem = BasePickerElement<Region>
 public final class FormRegionPickerItem: BaseFormPickerItem<Region> {
 
     internal init(preselectedValue: Region, selectableValues: [Region], style: FormTextItemStyle) {
-        super.init(preselectedValue: RegionPickerItem(identifier: preselectedValue.identifier, element: preselectedValue),
-                   selectableValues: selectableValues.map { RegionPickerItem(identifier: $0.identifier, element: $0) },
+        super.init(preselectedValue: Self.pickerElement(from: preselectedValue),
+                   selectableValues: selectableValues.map(Self.pickerElement),
                    style: style)
     }
 
+    public func updateData(_ regions: [Region]) {
+        assert(!regions.isEmpty)
+        selectableValues = regions.map(Self.pickerElement)
+        value = selectableValues[0]
```

Comment:
this is a bit dangerous :)

Reply:
But what choose do I have? 🤔

Reply:
maybe use `if let firstElement =  regions.first`

Reply:
I would use 
```
value = selectableValues.first
```

this way its safer

Reply:
`first ` is optional. Here we assume that array **is never empty**. If it is empty - this is serious error

Reply:
this method is not used